src/backend/TrafficCourts/Common/Utils/DateOnlyJsonConverter.cs

```diff
@@ -9,18 +10,19 @@
 
 namespace TrafficCourts.Common.Utils
 {
-    public class DateOnlyJsonConverter : JsonConverter<DateOnly>
-    {
-        private const string DateFormat = "yyyy-MM-dd";
+    //public class DateOnlyJsonConverter : JsonConverter<DateOnly>
```

Comment:
note required anymore, had commented out to validated these were not used anywhere.